must-gather/collection-scripts/gather_namespace_resources

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Expect base collection path as an argument
+BASE_COLLECTION_PATH=$1
+
+# Use PWD as base path if no argument is passed
+if [ "${BASE_COLLECTION_PATH}" = "" ]; then
+    BASE_COLLECTION_PATH=$(pwd)
+fi
+
+# Add general resources to list if necessary
+
+# Resource List
+resources=()
+# collect storagecluster resources
+
+# TODO: Re enable the collection of storagecluster via inspect command
+# resources+=(storageclusters)
+
+# collect OB/OBC resoureces
+resources+=(objectbucketclaims)
+resources+=(objectbuckets)
+
+# collection path for OC commands
+mkdir -p ${BASE_COLLECTION_PATH}/oc_output/
+
+# Command List
+commands=()
+
+# collect oc output of OC commands
+commands+=("get clusterversion -oyaml")
+commands+=("get pods -owide -n openshift-storage")
+commands+=("get nodes --show-labels")
+commands+=("describe nodes")
+commands+=("describe pods -n openshift-storage")
+commands+=("get pvc --all-namespaces")
+commands+=("get pv")
+commands+=("get infrastructures.config -oyaml")
+commands+=("get subscription -n openshift-storage")
+commands+=("get csv -n openshift-storage")
+commands+=("get sc")
+commands+=("get installplan -n openshift-storage")
+commands+=("get events -n openshift-storage")
+commands+=("get all -n openshift-storage")
+commands+=("clusterrole -n openshift-storage")
+commands+=("clusterrolebinding -n openshift-storage")
+commands+=("role -n openshift-storage")
+commands+=("rolebinding -n openshift-storage")
```

Comment:
Not a hard requirement, but maybe arrange related resources together?
Like StorageClass, PV, PVC in one block, nodes in other, pods in other etc.
Just for easier code nagivation.

Reply:
Use global constant for namespace.

Reply:
I can separate out namespced resources.